### PR TITLE
:bug: try to fix the bug in crf under tf 1.14

### PR DESCRIPTION
**Pull request by alexwwang**
To see if this works.

### CODE REVIEW

**Comment by BrikerMan** [comment]
I have used another CRF implement, it works, but gives me a wired accuracy and loss like this.

```
Epoch 1/50
20800/20864 [============================>.] - ETA: 0s - loss: 22.4886 - accuracy: 0.9280
epoch: 0 precision: 0.276740, recall: 0.281623, f1: 0.279161
20864/20864 [==============================] - 122s 6ms/sample - loss: 22.4417 - accuracy: 0.9281 - val_loss: 161.6057 - val_accuracy: 0.4200
Epoch 2/50
20800/20864 [============================>.] - ETA: 0s - loss: 6.4196 - accuracy: 0.9765
epoch: 1 precision: 0.571409, recall: 0.553882, f1: 0.562509
20864/20864 [==============================] - 114s 5ms/sample - loss: 6.4180 - accuracy: 0.9765 - val_loss: 158.0664 - val_accuracy: 0.6997
Epoch 3/50
20800/20864 [============================>.] - ETA: 0s - loss: 4.3281 - accuracy: 0.9828
epoch: 2 precision: 0.667961, recall: 0.653637, f1: 0.660722
20864/20864 [==============================] - 114s 5ms/sample - loss: 4.3292 - accuracy: 0.9828 - val_loss: 155.8536 - val_accuracy: 0.6192
Epoch 4/50
20800/20864 [============================>.] - ETA: 0s - loss: 3.3476 - accuracy: 0.9857
epoch: 3 precision: 0.707975, recall: 0.715662, f1: 0.711798
20864/20864 [==============================] - 115s 6ms/sample - loss: 3.3504 - accuracy: 0.9857 - val_loss: 154.1770 - val_accuracy: 0.6126
Epoch 5/50
20800/20864 [============================>.] - ETA: 0s - loss: 2.6810 - accuracy: 0.9879
epoch: 4 precision: 0.749690, recall: 0.737785, f1: 0.743690
20864/20864 [==============================] - 116s 6ms/sample - loss: 2.6836 - accuracy: 0.9879 - val_loss: 152.9230 - val_accuracy: 0.3055
Epoch 6/50
20800/20864 [============================>.] - ETA: 0s - loss: 2.1745 - accuracy: 0.9897
epoch: 5 precision: 0.799706, recall: 0.738056, f1: 0.767645
20864/20864 [==============================] - 115s 6ms/sample - loss: 2.1778 - accuracy: 0.9897 - val_loss: 151.9838 - val_accuracy: 0.0662
Epoch 7/50
20800/20864 [============================>.] - ETA: 0s - loss: 1.8038 - accuracy: 0.9911
epoch: 6 precision: 0.790598, recall: 0.778366, f1: 0.784434

```

**Comment by alexwwang** [comment]
weird indeed.

**Comment by BrikerMan** [comment]
> weird indeed.

Yep, but performance is great actually. F1 is increasing.

**Comment by alexwwang** [comment]
> > weird indeed.
> 
> Yep, but performance is great actually. F1 is increasing.

try to locate the loss function to see anything wrong?
but it's good news the performance promoted.

**Comment by BrikerMan** [comment]
I think we should stick to the new CRF layer, it's code more clean and simple.

**Comment by alexwwang** [comment]
> I think we should stick to the new CRF layer, it's code more clean and simple.

I agree. Where do you find it?

**Comment by BrikerMan** [comment]
https://github.com/search?q=tf.keras+crf&type=Code first one. But there are lots of the similiar implement.

**Comment by alexwwang** [comment]
thanks. ~
